src/content/en/2019/javascript.md

```diff
@@ -391,7 +387,7 @@ JavaScript continues to evolve as a language. A new version of the language stan
 
 With HTTP Archive, we can take a look at any newer API that is supported (or is about to be) and see how widespread its usage is. These APIs may already be used in browsers that support them _or_ with an accompanying polyfill to make sure they still work for every user.
 
-**How many sites use the following APIs?**
+How many sites use the following APIs?
```

Comment:
Same.

Reply:
Again will leave this one as it introduces the following list.